packages/@aws-cdk/aws-apigateway/README.md

```diff
@@ -1016,6 +1016,20 @@ const api = new apigateway.SpecRestApi(this, 'ExampleRestApi', {
 [`x-amazon-apigateway-endpoint-configuration`](https://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-endpoint-configuration.html) 
 in your openApi file. 
 
+### Permissions
+
+A resource policy can be passed as the prop or will be automatically created for the API upon the first call to
```

Comment:
Before stating this, add a couple of sentences around what these permissions mean or when they are commonly used. 

If APIGateway has this already documented, you can use those directly.